jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby-1.7-vs-9k/Jenkinsfile

```diff
@@ -27,11 +30,14 @@ pipeline.multipass_pipeline([
         ],
         [
                 job_name: 'oss-latest-jruby-9k',
```

Comment:
Do you know if our Jenkins could have collisions if we use the same `job_name` across different pipelines?  This one's name I guess is technically different than the `oss-latest-jruby9k` one in the `oss-puppetserver-latest-jruby9k` scenario but the two are pretty similar.  If pipeline job names can't collide, I think this is fine.  If not, though, we might name this one a little differently to make it less likely to collide with another job_name, like `oss-latest-jruby-9k-for-1.7-comparison` or something.  naming--